hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_constants.py

```diff
@@ -59,6 +67,9 @@
 GET_WALLET_BALANCE_PATH_URL = {
     LINEAR_MARKET: f"{REST_API_VERSION}/private/wallet/balance",
     NON_LINEAR_MARKET: f"{REST_API_VERSION}/private/wallet/balance"}
+SET_POSITION_MODE_URL = {
+    LINEAR_MARKET: "private/linear/position/switch-mode",
+    NON_LINEAR_MARKET: "private/linear/position/switch-mode"}
```

Comment:
The non linear market (called inverse perpetual by Bybit) does not seem to have an endpoint to set the position. Why are we using the same as for linear markets?

Reply:
You were correct. The situation was that there's no trading pair in this request, and the default value in get_rest_api_market_for_endpoint() in that case was non-linear market, therefore an URL for a non-linear market was requested. But since the correct URL was provided, the request was actually processed correctly.